proto/cosmos/evidence/genesis.proto

```diff
@@ -7,5 +7,6 @@ import "google/protobuf/any.proto";
 
 // GenesisState defines the evidence module's genesis state.
 message GenesisState {
-  repeated google.protobuf.Any evidence = 1;
+  // evidence defines all the evidences at genesis.
+  repeated google.protobuf.Any evidences = 1;
```

Comment:
changed field to plural, because it's repeated

Reply:
nit: evidence can be countable or uncountable

Reply:
ah, TIL. Will change it back then.